kernel/panic.c

```diff
@@ -212,7 +212,7 @@ void panic(const char *fmt, ...)
 	if (len && buf[len - 1] == '\n')
 		buf[len - 1] = '\0';
 
-	pr_emerg("Kernel panic - not syncing: %s\n", buf);
+	pr_emerg("Vibe check - not syncing: %s\n", buf);
```

Comment:
Not funny. Move on.

Reply:
What???

Reply:
The kernel has not passed the vibe check